test/MultiCursorTest.re

```diff
@@ -46,11 +56,11 @@ describe("Multi-cursor", ({describe, _}) => {
       let updates: ref(list(BufferUpdate.t)) = ref([]);
       let dispose = Buffer.onUpdate(upd => updates := [upd, ...updates^]);
 
-      let _ = Vim.input("i");
+      let _ = input("i");
```

Comment:
Possible accidental partial application. Not great in tests either, although definitely easier to track down here.